conduct-faq.html

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: "Code of Conduct: FAQ"
+---
+
+<div id="content">
+
+<h2>Code of Conduct: FAQ</h2>
+
+<p>
+Largely taken from <a href="https://www.djangoproject.com/conduct/faq/">django</a>. Thanks!
+</p>
+
+<p>
+This FAQ attempts to address common questions and concerns around the i3
+community's <a href="/conduct">Code of Conduct</a>. If you still have
+questions after reading it, please feel free to contact us at conduct AT i3wm DOT org.
```

Comment:
You’ll automatically get emails sent to that address (it’s a group).

If you want, I can set up an i3wm.org email account for you, but note that it’s not a pre-requisite for releases — I never used an i3wm.org address for releases. In any case, let’s continue this discussion outside of this pull request :).